nsock/src/Makefile.in

```diff
@@ -86,12 +86,12 @@ config.h: stamp-h
 stamp-h: config.h.in config.status
 	./config.status
 
-Makefile: Makefile.in config.status
+Makefile: $(srcdir)/Makefile.in config.status
 	./config.status
 
-config.status: configure
+config.status: $(srcdir)/configure
 	./config.status --recheck
 
 makefile.dep:
 	$(CC) -MM $(CPPFLAGS) $(SRCS) > $@
-include makefile.dep
+-include makefile.dep
```

Comment:
This (the minus before include) is actually to silent the warning message sent by the Makefile:

1. the makefile is first read entirely, and the include produce a warning because makefile.dep has not been created yet
2. the makefile.dep is created
3. the makefile is launched from the default target rule 

Thus, with a minus, it will just silent the first attempt to "include makefile.dep". This may be needed in lots of other Makefile.in files.